tutorial1.md

```diff
@@ -0,0 +1,299 @@
+[kubeletdoc]: https://godoc.org/k8s.io/kubelet/config/v1beta1#KubeletConfiguration
+[kubeletauth]: https://godoc.org/k8s.io/kubelet/config/v1beta1#KubeletAuthentication
+
+```yaml
+anonymous:
+  enabled: false
+webhook:
+  enabled: true
+  cacheTTL: "2m"
+```
+
+Let's override that:
+
+```yaml
+kind: KubeletConfiguration
+apiVersion: kubelet.config.k8s.io/v1beta1
+authentication:
+  webhook:
+    enabled: false
+  anonymous:
+    enabled: true
+authorization:
+  mode: AlwaysAllow
+```
+
+authentication and authorization are different.
+authentication answers "who are you," and authorization answers "are you allowed."
+To get the server to work in standalone mode, we have to give answers to both those questions.
+
+If all's gone well, you should see some log lines that look like this:
+
+```console
+$ sudo journalctl --lines 10 --unit kubelet
+-- Logs begin at Fri 2019-03-22 19:46:26 UTC, end at Sat 2019-03-23 02:43:50 UTC. --
+Mar 23 02:42:17 k8s-tutorial kubelet[24818]: I0323 02:42:17.374717   24818 kubelet_node_status.go:278] Setting node annotation to enable volume controller attach/detach
+Mar 23 02:42:27 k8s-tutorial kubelet[24818]: I0323 02:42:27.388418   24818 kubelet_node_status.go:278] Setting node annotation to enable volume controller attach/detach
+```
+
+Success! But the obvious question is: what can we _do_ with our lonely kubelet?
+
+## Running Static pods
+
+We're going to need a payload.
+This can be anything we want, but let's try [simpleservice][simpleservice].
+
+First, we need to modify our kubelet manifest to run static pods:
+
+```yaml
+kind: KubeletConfiguration
+apiVersion: kubelet.config.k8s.io/v1beta1
+authentication:
+  webhook:
+    enabled: false
+  anonymous:
+    enabled: true
+authorization:
+  mode: AlwaysAllow
+staticPodPath: /etc/kubernetes/pods/
+```
+
+And restart the process:
+
+A container isn't enough to run this, so we'll need a pod spec as well.
+The `pod` is something you may find yourself writing, and there's even an [API page][podapi]!
+
+[simpleservice]: https://hub.docker.com/r/mhausenblas/simpleservice/
+[podapi]: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#pod-v1-core
+
+You should end up with something like this:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: myapp-pod
+  labels:
+    app: myapp
+spec:
+  containers:
+  - name: myapp-container
+    image: mhausenblas/simpleservice:0.5.0
+    ports:
+    - containerPort: 8080
+    env:
+    - name: PORT0
+      value: "8080"
+```
+
+Kubelet will look for that file and start the pod. You can see this in the output from docker:
+
+```console
+$ sudo docker ps
+CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS              PORTS               NAMES
+f96c7c333214        mhausenblas/simpleservice   "python ./simpleserv…"   7 minutes ago       Up 7 minutes                            k8s_myapp-container_myapp-pod-k8s-tutorial_default_7a32a55f16cc86f3976f4b8e2ee88408_0
```

Comment:
the pause container shows up right away but the other simpleservice container takes some time to download and start up.